nemo/core/classes/modelPT.py

```diff
@@ -325,6 +346,169 @@ def test_dataloader(self):
     def test_step(self, batch, batch_ix):
         pass
 
+    def validation_epoch_end(
+        self, outputs: Union[List[Dict[str, torch.Tensor]], List[List[Dict[str, torch.Tensor]]]]
+    ) -> Optional[Dict[str, Dict[str, torch.Tensor]]]:
```

Comment:
The outer list corresponded to the data loaders index in the second case. This code presents a unified interface for multi* methods - they will only ever have to deal with the 1 data loader case. This instead will loop over 0/1/more than 1 dataset and dispatch to multi* method so that the end user can write code for just 1 data loader and have the support of any number of datasets.